lib/ConvexHull.js

```diff
@@ -42,6 +41,42 @@ var ConvexHull = ( function () {
 
 	Object.assign( ConvexHull.prototype, {
 
+		collectFacesAndVertices: function () {
+			// Get face vertex indices.
+			// These are the indices of all input vertices, including vertices inside the convex hull.
+			const faceIndices = this.faces.map((f) => f.collectIndices());
+
+			// Get distinct vertex indices referenced by convex hull faces, in ascending order.
```

Comment:
It seems like the comments and the code don't quite match here, if `faceIndices` includes interior vertices, and `referencedFaceIndices` is a sorted list of unique vertices in `faceIndices`, it seems like this includes interior vertices too?

Reply:
Updated to this now:

```ts
// Get face vertex indices.
// These indices reference the array positions of all input vertices, which include vertices inside the convex hull.
// These will be re-mapped to the convex hull's vertices, which exclude interior vertices.
const faceIndices = this.faces.map((f) => f.collectIndices());

// ...
```

Let me know if this makes sense!

Reply:
Yes, thanks!